contentcuration/contentcuration/static/js/edit_channel/file_upload/views.js

```diff
@@ -580,12 +586,143 @@ var FormatSlot = BaseViews.BaseListNodeItemView.extend({
         this.containing_list_view.update_metadata();
     },
     remove_item:function(){
-        this.containing_list_view.set_file_format(null, this.model, this.file);
-        this.file = null;
-        this.render();
+        if(this.language_view){
```

Comment:
Handle removing item by adding language back to dropdown and setting file as being unassigned